client/app/bundles/Zoon/components/WorkflowDiagram.jsx

```diff
@@ -27,7 +27,11 @@ const Switch = ({ value, onChange }) => (
 )
 
 const SelectedItem = ({ item, sharedProps }) => (
-  <div className="module-list__list__item">
+  <div
+    className="module-list__list__item"
+    onMouseEnter={() => sharedProps.wantsModuleShown(item.id)}
```

Comment:
`wantsModuleShown` is a weird name to me, as it indicates controlling the parent behavior.
I'd go with `onModuleHover` or something like that, and let the parent/callee do whatever it wishes on hover.